src/main/java/frc/robot/maps/RobotMap.java

```diff
@@ -7,18 +7,26 @@
 
 import com.chopshop166.chopshoplib.maps.RobotMapFor;
 // $Imports$
+import frc.robot.maps.subsystems.ArmRotateMap;
 import frc.robot.maps.subsystems.SwerveDriveMap;
 
+
 @RobotMapFor("Default")
 public class RobotMap {
     // $Maps$
+    private ArmRotateMap armRotateMap = new ArmRotateMap();
```

Comment:
We should just make this directly in the function call (yes I know the tool generated it this way, yes I want it changed)